polaris-react/src/components/ActionMenu/ActionMenu.scss

```diff
@@ -1,11 +1,14 @@
 .ActionMenu {
   // Needed for JS calculations
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   width: 100%;
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   display: flex;
+  // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
   justify-content: flex-end;
 
   @media print {
-    // stylelint-disable-next-line declaration-no-important
+    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
```

Comment:
since the line below breaks both the existing `declaration-no-important` rule and new stylelint-polaris rules, I strip out the rule specification and disable for all